src/index.js

```diff
@@ -1,11 +1,10 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
-import 'semantic-ui-less/semantic.less'
+import React from "react";
```

Comment:
Can we stick to the single quote for string. Also can we do a basic setup for linting.
I would recomment to follow airbnb rules.. The .eslintrc file look something like this

```
{
  "extends": "airbnb",
  "parserOptions": {
    "sourceType": "module",
    "ecmaFeatures": {
      "jsx": true
    }
  },
  "env": {
    "browser": true,
    "commonjs": true,
    "es6": true,
    "jquery": true,
    "jest": true
  },
  "rules": {
    "react/jsx-filename-extension": 0,
    "jsx-a11y/label-has-for": 0,
    "jsx-a11y/label-has-associated-control": 0
  }
}
```